cfme/configure/settings.py

```diff
@@ -1,424 +1,312 @@
-            'hours': updates.get('hours'),
-            'timezone': updates.get('timezone'),
-        })
-        if changed:
-            view.timeprofile_form.save_edit_button.click()
-            view.flash.assert_message(
-                'Time Profile "{}" was saved'.format(updates.get('description', self.description)))
+    containers_providers = ViewButtonGroup('Containers', 'Containers Providers')
+    container_nodes = ViewButtonGroup('Containers', 'Nodes')
+    container_pods = ViewButtonGroup('Containers', 'Pods')
+    container_services = ViewButtonGroup('Containers', 'Services')
+    container_routes = ViewButtonGroup('Containers', 'Routes')
+    container_containers = ViewButtonGroup('Containers', 'Containers')
+    container_projects = ViewButtonGroup('Containers', 'Projects')
+    container_replicators = ViewButtonGroup('Containers', 'Replicators')
+    container_images = ViewButtonGroup('Containers', 'Images')
+    container_image_registries = ViewButtonGroup('Containers', 'Image Registries')
+    container_builds = ViewButtonGroup('Containers', 'Builds')
+    container_volumes = ViewButtonGroup('Containers', 'Volumes')
+    container_templates = ViewButtonGroup('Containers', 'Templates')
+    vm_visibility = BootstrapSwitch('display_vms')
+    save = Button('Save')
 
-    def copy(self, name=None):
-        view = navigate_to(self, 'All')
-        rows = view.timeprofile_form.table
-        for row in rows:
-            if row.description.text == self.description:
-                row[0].check()
-        view.timeprofile_form.configuration.item_select('Copy selected Time Profile')
-        if name is not None:
-            new_timeprofile = Timeprofile(description=name, scope=self.scope)
-            changed = view.timeprofile_form.fill({
-                'description': name,
-                'scope': self.scope,
-            })
-        else:
-            new_timeprofile = Timeprofile(description="{} copy".format(self.description),
-                                          scope=self.scope)
-            changed = view.timeprofile_form.fill({
-                'description': "{} copy".format(self.description),
-                'scope': self.scope,
-            })
 
-        if changed:
-            view.timeprofile_form.save_button.click()
-        return new_timeprofile
-
-    def delete(self):
-        view = navigate_to(self, 'All')
-        rows = view.timeprofile_form.table
-        for row in rows:
-            if row.description.text == self.description:
-                row[0].check()
-        view.timeprofile_form.configuration.item_select("Delete selected "
-                                                        "Time Profiles", handle_alert=True)
+class DefaultFiltersForm(View):
+    tree = CheckableBootstrapTreeview('df_treebox')
+    save = Button('Save')
+
 
+class TimeProfilesView(View):
+    table = Table("//div[@id='main_div']//table")
+    flash = FlashMessages('.//div[@id="flash_msg_div"]'
```

Comment:
**Question**: Is this flash element is not the same as we have on BaseLoggedInPage? If the same please remove as we can use one from BaseLoggedInPage

Reply:
Yes, this was done before the FlashMessages change.